nss-ice-phantoms/BangazonAPI
# Jb active false shows no orders

# Description
Description of changes.
Added to customerController .... now in browser url one can type `http://localhost:5000/api/customers/?_active=false` and see customers who do not have any active orders


## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout jb-activeFalseShowsNoOrders`
1.  run your api server
1. be sure that your database contains customers who do not have an order (their customerId should not be visible in [order] table) or be aware that you'll receive an empty json if all your customers do have orders
1. Type `http://localhost:5000/api/customers/?active=_false` in your browser or postman for `GET`.
1. you should see in return a json with customer objects.  These should be only and all customers who do not have orders.
1. Then `http://localhost:5000/api/customers/?active=_true` to see customers that have active orders


# Checklist:
